engine/language-server/src/main/scala/org/enso/languageserver/search/SuggestionsHandler.scala

```diff
@@ -493,6 +465,13 @@ final class SuggestionsHandler(
       updates.foreach(sessionRouter ! _)
       context.become(initialized(name, graph, clients, state))
 
+    case libraryLoaded: Api.LibraryLoaded =>
+      logger.debug(
```

Comment:
@4e6's idea is to deliver the cached `suggestion.json` here. However I haven't hit a breakpoint placed here yet. I am not sure who's supposed to deliver this message?

Reply:
One has to subscribe to Akka event bus to get the notification - f480e03f244c6716bf20ab2b13fbdb8076c89056

But now there is another problem: The `SuggestionSerializationManager` is in runtime, while this code is in `language-server` - how shall these two talk to each other? Send another message? Asynchronous? It is convoluted...